python/python-core/syntax-and-numerical-operators/execution-order-of-arithmetic-operators.md

```diff
@@ -52,11 +52,10 @@ Here is the order of execution for Python's operators (highest to lowest):
 What’s the output of the following code:
 
 ```bash
-2 + 7 * 2 // 8
+>>> 2 + 7 * 2 // 8
+>>> ???
```

Comment:
Just for awareness, why add `>>>`?

Reply:
forgot to leave a comment here :sweat_smile: 

I'm not sold on this approach, so I think we can remove `>>>`

the idea behind is that you would type all that command in the interpreter

Reply:
so we would simulate that feeling, but now that you pointed it out it might be better to remove them